File: 01_Leetcode_FAANG/Rajanikant_Pattnaik/concatenation_of_array.cpp
class Solution {
public:
    vector<int> getConcatenation(vector<int>& nums) {
     int n=nums.size();
     vector<int>ans;
     for(int i=0;i<2*n;i++){
       ans.push_back(nums[i%n]);   
     }
    return ans;
    }
};

File: 01_Leetcode_FAANG/Swagat_Shekhar/Concatenation_of_Array.java
class Solution {
    public int[] getConcatenation(int[] nums) {
        
          int n = nums.length;
        int ans[] = new int[2*n];
        for(int i =0; i< ans.length;i++)
        {
            ans[i] = nums[i%n];
        }
        return ans;
    }
        
    }
